getting-started/FAQ.md

```diff
@@ -96,3 +96,7 @@ KubeArmor defines 3 policy actions: Allow, Block and Audit.
 
 If Block policy is used and there are no supported enforcement mechanism on the platform then the policy enforcement wouldn't be observed. But we will still be able to see the observability data for the applied Block policy, which can help us in identifying any suspicious activity.
 </details>
+
+<details><summary><h4>Why can't I find the KubeArmor node when using `make run`?</h4></summary>
+ If you have followed all necessary steps, it may be due to a mismatch between the hostname(can be obatined by running `hostname`) and the hostname shown under  Name  section in the output of `kubectl get nodes -o wide`. This can happen when the hostname has different letter casing from the name shown in the output.And, it happens because kubernetes lowers the casing.  To resolve this, change the hostname to a some lowercase name with the command `hostnamectl set-hostname` <lowercase-hostname>.
```

Comment:
> If you have followed all necessary steps, it may be due to a mismatch between the hostname(can be obatined by running `hostname`) and the hostname shown under  Name  section in the output of `kubectl get nodes -o wide`. This can happen when the hostname has different letter casing from the name shown in the output.And, it happens because kubernetes lowers the casing.  To resolve this, change the hostname to a some lowercase name with the command `hostnamectl set-hostname` <lowercase-hostname>.

If you have followed all necessary steps, the discrepancy between the actual hostname (obtained by running `hostname`) and the hostname used by Kubernetes (under `kubectl get nodes -o wide`) may be due to case sensitivity issue. K8s converts the hostname to lowercase, which results in a mismatch with the actual hostname. 
To resolve this, change the hostname to a lowercase version using the command `hostnamectl set-hostname <lowercase-hostname>`